src/utils.zig

```diff
@@ -29,6 +29,15 @@ pub fn sleep(nanoseconds: u64) void {
     }
 }
 
+pub fn clock_gettime_monotonic() u64 {
+    const ts = std.posix.clock_gettime(std.posix.clockid_t.MONOTONIC) catch unreachable;
+
+    const s = @as(u64, @intCast(ts.sec)) * std.time.ns_per_s;
+    const nsec: u64 = @intCast(ts.nsec);
+
+    return s + nsec;
```

Comment:
I think it's worth noting down the max timestamp that can be represented in this nsec only format. 
I get 2554-07-21T23:34:33.709551615, I let you do the math as well to confirm. But we really need to note this down here so we'll be able to discard this as a potential source of bugs later on

Reply:
idk if it's too late but keeping the same unix format s/ns would maybe be simpler down the line